DOCUMENTATION.md

```diff
@@ -29,6 +30,12 @@ This module implements all models for Resin Python SDK.
 ## Application
 
 This class implements application model for Resin Python SDK.
+
+Due to API changes, the returned Application object schema has changed. Here are the formats of the old and new returned objects.
+
+The old returned object's properties: `__metadata, actor, app_name, application, commit, device_type, git_repository, id, support_expiry_date, user, version, should_track_latest_release`.
```

Comment:
List should be:
__metadata, actor, app_name, application, commit, device_type, git_repository, id, should_track_latest_release, support_expiry_date, user, version